github-actions/verify-pr-title/package.json

```diff
@@ -8,24 +8,37 @@
 		"build": "tsc",
 		"lint": "eslint src/**/*.ts",
 		"package": "npx ncc build",
-		"test": "mocha \"./test/**/*.test.ts\"",
+		"test": "jest",
```

Comment:
For whatever reason, Mocha _really_ didn't like running inside a subfolder where the parent folder was using Jest. So I just went ahead and converted the tests to Jest.